### DISCUSSION TITLE
Is triggering the ChangeContainerOperation available in GLSP 2.0?

### DISCUSSION THREAD

**Discussion by quratulain-ali** [eclipse-glsp/glsp#1228] [General]
I have seen that triggering the ChangeContainerOperation wasn't available in the base framework earlier as pointed [here](https://github.com/eclipse-glsp/glsp/discussions/731). Is this feature available in GLSP 2.0?

**Comment by tortmayr** [comment]
Hi @quratulain-ali,

no unfortunately the `ChangeContainerOperation` is currently still only available on a protocol level. i.e. it is defined in the protocol but there is no tool support for it on the client side.

I have created https://github.com/eclipse-glsp/glsp/issues/1235 to track this.
Generic support for container changes as part of the default `ChangeBoundsTool` definitely make sense, so maybe we can plan that in for one of the next releases.
